docs/recipes/lighthouse-plugin-example/readme.md

```diff
@@ -11,9 +11,9 @@ Run the following to start of with the recipe as a template:
 
 ```sh
 mkdir lighthouse-plugin-example && cd lighthouse-plugin-example
-curl -L https://github.com/GoogleChrome/lighthouse/archive/master.zip | tar -xzv
-mv lighthouse-master/docs/recipes/lighthouse-plugin-example/* ./
-rm -rf lighthouse-master
+curl -L https://github.com/GoogleChrome/lighthouse/archive/main.zip | tar -xzv
+mv lighthouse-main/docs/recipes/lighthouse-plugin-example/* ./
+rm -rf lighthouse-main
```

Comment:
Did you confirm this is how the zip is created in github

Reply:
well.. i have now. :)

```sh
curl -L https://github.com/GoogleChrome/lighthouse/archive/master-to-main.zip | tar -xzv
# ...
file lighthouse-master-to-main/ # lighthouse-master-to-main/: directory
```